gcp_variant_transforms/libs/bigquery_vcf_schema_converter.py

```diff
@@ -203,7 +203,8 @@ def generate_header_fields_from_schema(schema, allow_incompatible_schema=False):
   infos = OrderedDict()  # type: OrderedDict[str, _Info]
   formats = OrderedDict()  # type: OrderedDict[str, _Format]
   for field in schema.fields:
-    if field.name in _NON_INFO_OR_FORMAT_CONSTANT_FIELDS:
+    if (field.type == bigquery_util.TableFieldConstants.TYPE_DATE or
```

Comment:
Consider doing a "whitelist" rather than a "blacklist". In the current way ("blacklist" approach) you'd also need to include other types like TIMESTAMP (or any new field that gets added later). I think you can create a helper method like `_get_supported_bigquery_schema_types` that returns all supported types (ideally, we'd use an enum to return this programmatically, but ok to hard code it for now).